django_icons/renderers/image.py

```diff
@@ -0,0 +1,73 @@
+import os
+
+from django.conf import settings
+from django.forms.utils import flatatt
+from django.utils.html import format_html
+from django.utils.safestring import mark_safe
+
+from django_icons.css import merge_css_text
+from django_icons.renderers.base import BaseRenderer
+
+
+class ImageRenderer(BaseRenderer):
+    """
+    Render an icon from a local file
+
+    <img src="/static/icons/icon.png'" class="icon icon-{name}" %}">
```

Comment:
It would be nice if you could include the template tag to produce a certain output.